sys/include/auto_init.h

```diff
@@ -8,22 +8,55 @@
  * directory for more details.
  */
 
-
 /**
- * @defgroup    sys_autoinit Auto-init
+ * @defgroup    sys_auto_init Auto-init
  * @ingroup     sys
  * @brief       Auto initialize modules
- * @note        This feature can be used by any application by adding auto_init to
- *              USEMODULE in the application's Makefile. auto_init will initialize
- *              any other included module that does not require a parameter in
- *              its init function, i.e. if the prototype looks like this: void
- *              MODULE_init(void). Most timer modules or simple drivers can be
- *              initialized by auto_init. The modules will be initialized in
- *              the context of the main thread right before the main function
- *              gets called. Be aware that most modules expect to be
- *              initialized only once, so do not call a module's init function
- *              when using auto_init unless you know what you're doing.
  *
+ * This feature can be enabled in any application by adding the `auto_init`
+ * module to the application Makefile:
+ *
+ * ```mk
```

Comment:
This syntax doesn't work in Doxygen. Try

```markdown
~~~~~~~~~~~~~~~~~~~~~~~ {.mk}
USEMODULE += auto_init
~~~~~~~~~~~~~~~~~~~~~~~
```

instead if you like to enable syntax highlighting, otherwise remove the `mk`.

Reply:
removed the commit related to that change. Will open a dedicated PR for that later.